pom.xml

```diff
@@ -569,7 +569,6 @@
                         <dependency>
                           <groupId>com.puppycrawl.tools</groupId>
                           <artifactId>checkstyle</artifactId>
-                          <version>${checkstyle.version}</version>
```

Comment:
I _think_ you can remove all `<dependencies>` altogether here, given that they are already defined in the pluginManagement section. This should just inherit it.